book.py

```diff
@@ -1,12 +1,12 @@
 class Book:
     def __init__(self, title, author, pages):
-        self.title = title
+        self.title = titl
         self.author = author
         self.pages = pages
 
     def summary(self):
```

Comment:
Comment: The change in the `__init__` method where `self.title` is assigned to `titl` instead of `title` is a typographical error. This will cause the `summary` method to reference an undefined attribute, leading to an AttributeError when attempting to access `self.title`. The `is_long_book` method's threshold change from 300 to 400 pages may also lead to inconsistency with existing documentation or user expectations, but the immediate issue is the incorrect assignment in the `__init__` method.